### PR TITLE
chore: clean up test

### PR DESCRIPTION
**Pull request by taka-oyama**

### CODE REVIEW

File: src/Colopl/Spanner/Schema/Blueprint.php [hunk 1]
```diff
@@ -131,7 +131,7 @@ public function floatArray($column)
 
     /**
      * @param string $column
-     * @param int $length
+     * @param int|string $length
```
Comment (tyabu12) [inline src/Colopl/Spanner/Schema/Blueprint.php:134]:
```suggestion
     * @param int|'MAX'$length
```

I think that `$length` accepts only 'MAX' as string.
https://phpstan.org/r/91c2b2b0-bbf1-4b66-b216-453c3bb73c7f

However, PhpStorm still cannot recognize string literal of PHPDoc, so I leave it up to you to take it in.

File: tests/Colopl/Spanner/Eloquent/ModelTest.php [hunk 2]
```diff
@@ -21,13 +21,20 @@
...
+use Illuminate\Database\Eloquent\Relations\HasOne;
 use Illuminate\Routing\Middleware\SubstituteBindings;
 use Illuminate\Routing\Router;
 use Illuminate\Support\Carbon;
 
 /**
  * @property string $userId
  * @property string $name
+ * @property UserInfo $info
+ * @property Collection<UserItem> $items
```
Comment (tyabu12) [inline tests/Colopl/Spanner/Eloquent/ModelTest.php:37]:
```suggestion
 * @property Collection<int, UserItem> $items
```

File: tests/Colopl/Spanner/Eloquent/ModelTest.php [hunk 3]
```diff
@@ -89,6 +98,7 @@ public function user()
 /**
  * @property string $itemId
  * @property string $name
+ * @property Collection<Tag> $tags
```
Comment (tyabu12) [inline tests/Colopl/Spanner/Eloquent/ModelTest.php:101]:
```suggestion
 * @property Collection<int, Tag> $tags
```

File: tests/Colopl/Spanner/TransactionTest.php [hunk 4]
```diff
@@ -260,42 +252,34 @@ public function testLockTimeout(): void
...
-        } catch (AbortedException $ex) {}
+            $conn->transaction(fn() => throw new AbortedException('abort'));
+        } catch (AbortedException) {
+
+        }
 
-        $this->assertEquals(0, $committedCount);
-        $this->assertEquals(10, $rolledBackCount);
+        Event::assertDispatchedTimes(TransactionCommitted::class, 0);
+        Event::assertDispatchedTimes(TransactionRolledBack::class, 10);
```
Comment (tyabu12) [inline tests/Colopl/Spanner/TransactionTest.php:283]:
```suggestion
        Event::assertDispatchedTimes(TransactionRolledBack::class, Database::MAX_RETRIES));
```

`use Google\Cloud\Spanner\Database` is also needed.

File: src/Colopl/Spanner/Schema/Blueprint.php (hunk 1 above)
Comment (taka-oyama) [reply src/Colopl/Spanner/Schema/Blueprint.php:134]: Thanks, but IDE support is crucial. Let's revisit when PhpStorm supports this.

**Comment by tyabu12** [comment]
LGTM

**Pull request merged**